README.md

```diff
@@ -304,6 +314,8 @@ PostGIS polygon type must be referenced as `st_polygon`.
 
 ### Configuring ActiveRecord
 
+_NOTE: In version 7.0.0 changes were made so that the attributes in the second argument of `register` will be camel_cased, which may break existing configurations. See [this commit](https://github.com/rgeo/activerecord-postgis-adapter/pull/325/commits/075d8db036c29b88c0e13234affef260d262e76f) for more details._
```

Comment:
I do not find this statment very clear:

- there is no `camelCase` but rather only `snake_case`
- I'm no sure the commit really helps out, maybe a short example would be better ?

Reply:
Good catch! I've updated and added an example.